.infrastructure/7_ssm.tf

```diff
@@ -0,0 +1,38 @@
+resource "random_password" "database_password" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "aws_ssm_parameter" "db_password" { ### SECRET AWS EVENTUALMENTE DA CREARE A MANO PER NON METTERLA SU REPO PUBBLICO
+  name = "/website-strapi/db_password"       ### CONTENUTA ANCHE NEL FILE .env DA NON RENDERE VISIBILE
+  type = "SecureString"
+  ##value = var.rds_cluster_password
+  value = random_password.database_password.result
+}
+
+resource "random_password" "jws_secret" {
```

Comment:
This looks a lot like the `random_password.database_password` defined above. Honestly I do not know what the `random_password` resource do: is it a generator? Is it possible to avoid duplication and use just one `random_password` resource?

Reply:
It is a password generator that cannot be viewed even in output. To be safe I carried out a test and it is necessary to instantiate 2 times, it is not possible to use the same random_password resource. It returns the same value if called N times